src/nupic/engine/__init__.py

```diff
@@ -721,10 +721,19 @@ def save(self, *args, **kwargs):
       raise TypeError("Save path must be of type {}.".format(str))
     engine.Network.save(self, *args, **kwargs)
 
-  def inspect(self):
-    """Launch a GUI inpector to inspect the network"""
-    from nupic.analysis import inspect
-    inspect(self)
+  def getRegion(self, regionClass):
```

Comment:
There can be multiple regions of the same type in a network so this should return a sequence (empty if none match). Rename to `getRegionsByType`?